### ISSUE TITLE
config

### ISSUE THREAD

**Issue by zhujiem** [xue-pai/iFlyteck2022_CTR_Challenge_Baseline#2]
base_config: ../config/
base_expid: MLPX_base
dataset_id: avazu_x1

dataset_config:
    avazu_x1:
        data_root: ../data/Avazu/
        data_format: csv
        train_data: ../data/Avazu/Avazu_x1/train.csv
        valid_data: ../data/Avazu/Avazu_x1/valid.csv
        test_data: ../data/Avazu/Avazu_x1/test.csv
        min_categr_count: 1
        feature_cols:
            - {name: ["feat_1", "feat_2", "feat_3", "feat_4", "feat_5", "feat_6", "feat_7", "feat_8", "feat_9", "feat_10",
                      "feat_11", "feat_12", "feat_13", "feat_14", "feat_15", "feat_16", "feat_17", "feat_18", "feat_19", 
                      "feat_20", "feat_21", "feat_22"],
               active: True, dtype: float, type: categorical}
        label_col: {name: label, dtype: float}

tuner_space:
    model_root: './Avazu/MLPX_avazu_x1/'
    embedding_dim: 10
    embedding_regularizer: 5.e-2
    dnn1_batch_norm: True
    dnn2_batch_norm: True
    dnn1_hidden_units: [[400, 400, 400]]
    dnn2_hidden_units: [[800]]
    dnn1_dropout: [0.4, 0.3]
    dnn2_dropout: [0.2, 0.3]
    use_feature_selection: True
    selection_hidden_units: [[1000]]
    bilinear_group: [1, 2, 5, 10, 20, 50]
    learning_rate: 1.e-3
    batch_size: 4096
    seed: 2021
    monitor: 'AUC'
    monitor_mode: 'max'
    metrics: [['AUC', 'logloss']]
    verbose: 0

**Comment by zhujiem** [comment]
base_config: ../config/
base_expid: MLPX_base
dataset_id: frappe_x1

dataset_config:
    frappe_x1:
        data_root: ../data/Frappe/
        data_format: csv
        train_data: ../data/Frappe/Frappe_x1/train.csv
        valid_data: ../data/Frappe/Frappe_x1/valid.csv
        test_data: ../data/Frappe/Frappe_x1/test.csv
        min_categr_count: 1
        feature_cols:
            - {name: "user", active: True, dtype: float, type: categorical, source: user}
            - {name: [ "daytime", "weekday", "isweekend", "weather", "homework", "country", "city", "cost", "item"], active: True, dtype: float, type: categorical}
        label_col: {name: label, dtype: float}

tuner_space:
    model_root: './Frappe/MLPX_frappe_x1/'
    embedding_dim: 10
    embedding_regularizer: 5.e-2
    dnn1_batch_norm: True
    dnn2_batch_norm: True
    dnn1_hidden_units: [[400]]
    dnn2_hidden_units: [[100], [150]]
    dnn1_dropout: [0.4, 0.5]
    dnn2_dropout: [0.4, 0.5]
    use_feature_selection: True
    selection_hidden_units: [[800], [1000]]
    bilinear_group: 1
    learning_rate: 1.e-3
    batch_size: 4096
    seed: 2021
    monitor: 'AUC'
    monitor_mode: 'max'
    metrics: [['AUC', 'logloss']]

**Comment by zhujiem** [comment]
base_config: ../config/
base_expid: MLPX_base
dataset_id: movielenslatest_x1

dataset_config:
    movielenslatest_x1:
        data_root: ../data/Movielens/
        data_format: csv
        train_data: ../data/Movielens/MovielensLatest_x1/train.csv
        valid_data: ../data/Movielens/MovielensLatest_x1/valid.csv
        test_data: ../data/Movielens/MovielensLatest_x1/test.csv
        min_categr_count: 1
        feature_cols:
            - {name: "user_id", active: True, dtype: float, type: categorical}
            - {name: ["item_id", "tag_id"], active: True, dtype: float, type: categorical}
        label_col: {name: label, dtype: float}

tuner_space:
    model_root: './Movielens/MLPX_movielenslatest_x1/'
    embedding_dim: 10
    embedding_regularizer: 1.e-2
    dnn1_batch_norm: True
    dnn2_batch_norm: True
    dnn1_hidden_units: [[400]]
    dnn2_hidden_units: [[800]]
    dnn1_dropout: [0.4, 0.5]
    dnn2_dropout: [0.2, 0.3]
    use_feature_selection: True
    selection_hidden_units: [[800], [1000], [500]]
    bilinear_group: 10
    learning_rate: 1.e-3
    batch_size: 4096
    seed: 2021
    monitor: 'AUC'
    monitor_mode: 'max'
    metrics: [['AUC', 'logloss']]
